main.js

```diff
@@ -0,0 +1,161 @@
+document.querySelector('.search').addEventListener('click', () => {
+  const location = document.querySelector('#search-query').value;
+
+  document.querySelector('#search-query').value = '';
+
+  fetchCurrentConditions(location);
+  fetchFiveDayForecast(location);
+
+  
+});
+
+const fetchCurrentConditions = (location) => {
+  const url = `https://api.openweathermap.org/data/2.5/weather?q=${location}&units=imperial&appid=79a2d684772528c31363d342e6db9789`;
+
+  fetch(url, {
+    method: 'GET',
+    dataType: 'json'
+  })
+    .then(data => data.json())
+    .then(data => addCurrentConditions(data));
+}
+
+const fetchFiveDayForecast = (location) => {
+  const urlForecast = `https://api.openweathermap.org/data/2.5/forecast?q=${location}&units=imperial&cnt=40&appid=79a2d684772528c31363d342e6db9789`; 
+
+  fetch(urlForecast, {
+    method: 'GET',
+    dataType: 'json'
+  })
+    .then(dataForecast => dataForecast.json())
+    .then(dataForecast => addFiveDayForecast(dataForecast));
+}
+
+
+const addCurrentConditions = (data) => {
```

Comment:
data is a very generic name - would recommend using something more descriptive to the type of data